draft-ietf-quic-qpack.md

```diff
@@ -1035,17 +1038,19 @@ values that are not to be put at risk by compressing them (see Section 7.1 of
 ~~~~~~~~~~
 {: title="Literal Header Field With Name Reference"}
 
-For entries in the static table or in the dynamic table with an absolute index
-less than the Base, the header field name is represented using the relative
-index of that entry, which is represented as an integer with a 4-bit prefix (see
-Section 5.1 of [RFC7541]). The `S` bit indicates whether the reference is to the
-static (S=1) or dynamic (S=0) table.
+The fourth (`S`) bit indicates whether the reference is to the static (S=1) or
+dynamic (S=0) table.  Finally, the header field name is represented using the
+static or relative index of that entry, which is represented as an integer with
```

Comment:
We define relative, absolute, and post-base indices.  We talk about "static table index" and "index into the static table".  "Static table index" is used on line 656 of the current doc.  This use of "static index" is a new variation.  I suspect a reader could reasonably infer that they all mean the same thing -- obviously we all have -- but it might be nice to align the terminology and make sure it's defined somewhere.

Reply:
I think it's sufficient to just call them a relative index; the section defining that term describes relative to what in different contexts.